Makefile

```diff
@@ -47,7 +47,7 @@ test:
 test-integration:
 
 	@echo "==> Started integration tests"
-	@env GOCACHE=off go test -v -tags integration ./test/...
```

Comment:
why change this?

Reply:
Because go modules won't work if you turn of GOCACHE